sdk/src/wasm.ts

```diff
@@ -51,3 +55,50 @@ export {
     getOrInitConsensusVersionTestHeights,
     verifyFunctionExecution,
 } from "@provablehq/wasm/%%NETWORK%%.js";
+
+/**
+ * Initialize development mode by setting consensus version heights to [0..ConsensusVersion::latest()].
+ * 
+ * This function automatically sets up development consensus heights without requiring
+ * manual specification of the number of consensus versions. It should be called before
+ * initializing the thread pool when working with a local development network.
+ * 
+ * @returns An array of block heights at which each consensus version applies.
+ * 
+ * @example
+ * import { initDevMode, initThreadPool } from "@provablehq/sdk";
+ * 
+ * // Initialize dev mode before the thread pool
+ * initDevMode();
+ * await initThreadPool();
+ */
+export function initDevMode(): number[] {
+    return initDevModeWasm();
+}
```

Comment:
This extra wrapper doesn't add much, the wasm method itself is sufficient and can be exported directly.